[docs] clarification on loadChildren path

Hi there

sorry to bother again, it must be our project setup different from example. Main readme page, at [lazy loading section](https://github.com/brandonroberts/angular-router-loader#lazy-loading), says:

> [...] use `loadChildren` with a relative path to your lazy loaded angular module. 

Could you please elaborate more on "relative" to what? Should the lazy loaded module path be expressed as relative to the routing file where the route itself is defined? Or relative to the module file where the route will be "imported" with `RouterModule.forXxx()`?

To better express, say one has a structure like:

```
|-- compiled\
   |-- src\
      |-- app\
      |-- lazy\

|-- src\
   |-- app\
      |-- config\
         |-- app.routing.ts
      |-- app.module.ts

   |-- lazy\
      |-- lazy.module.ts

|-- package.json
```

then what version should `app.routing.ts` include?

```ts
// A: relative to routing file
export const routes: Routes = [
  { path: 'lazy', loadChildren: '../../lazy/lazy.module#LazyModule' }
];

// B: relative to module file
export const routes: Routes = [
  { path: 'lazy', loadChildren: '../lazy/lazy.module#LazyModule' }
];
```

Again, more than willing to submit a PR with a doc change if needed. Thanks!

The reason I'm asking, of course, is that I'm using a separate routing file placed 1 level below the `app.module.ts` file. From the compiled output, I can see the equivalent `app.module.ngFactory.ts` eventually includes somehow the route definitions and shows something like:

```ts
  get _ROUTES_88():any[] {
    if ((this.__ROUTES_88 == null)) { (this.__ROUTES_88 = [
        [{
          path: 'login',
          component: import83.LoginComponent,
          canActivate: [import62.LoginGuard]
        }
      ],
      [
        {
          path: '',
          loadChildren: '../../lazy/lazy.module#LazyModule',
          canLoad: [import66.AuthGuard]
        }
        ,
        {
          path: '**',
          component: import84.NotFoundComponent
        }

      ]

    ]
    ); }
    return this.__ROUTES_88;
  }
```

so basically the route definition has moved 1 level upward, but now that relative path is not good anymore. And of course build complains with (`codegen` is our `compiled` equivalent):

```
 ERROR in ../codegen/src/app/app.module.ngfactory.ts
 Module not found: Error: Can't resolve '..\..\lazy\lazy.module.ngfactory'
```

When I make the routing file a sibling of module file (and update the relative path accordingly as `loadChildren: '../lazy/lazy.module#LazyModule'`) build goes fine.

--- a/README.md
+++ b/README.md
@@ -41,8 +41,8 @@ export const routes: Routes = [
 
 **NOTE**: When specifying a relative path to lazy loaded module, one of the following two conditions *must* hold:
 
-* The lazy loaded route is defined in the same module file where it is added by `RouterModule.forXxx()`
-* The lazy loaded route is defined in a separate routing file, and that routing file is a sibling of module file (hence the routing file cannot be e.g. in a subdirectory).
+* The routes are defined in the same module file where it is imported with `RouterModule.forRoot` or `RouterModule.forChild`
+* The routes are defined in a separate routing file, and that routing file is a sibling of module file.
 
 ## Synchronous Loading
 

--- a/README.md
+++ b/README.md
@@ -39,6 +39,11 @@ export const routes: Routes = [
 ];
 ```
 
+**NOTE**: When specifying a relative path to lazy loaded module, one of the following two conditions *must* hold:
+
+* The lazy loaded route is defined in the same module file where it is added by `RouterModule.forXxx()`
+* The lazy loaded route is defined in a separate routing file, and that routing file is a sibling of module file (hence the routing file cannot be e.g. in a subdirectory).
+
 ## Synchronous Loading
 
 For synchronous module loading, add the `sync=true` as a query string value to your `loadChildren` string. The module will be included in your bundle and not lazy-loaded.

